### DISCUSSION TITLE
Note about the old name

### DISCUSSION THREAD

**Discussion by AlexWayfer** [wwmm/easyeffects#996] [General]
Hi.

It's OK to change the name (yeah, we have no deal with PulseAudio now I guess), but I suggest to add a note somewhere (repo description or README) about former PulseEffects name. It can be helpful for correct search results, old users and new users from out-of-dated external links.

**Comment by wwmm** [comment]
I agree. A few months ago I thought about that but I forgot to do it.

**Comment by AlexWayfer** [comment]
Also, I guess, we can drop `pulseaudio` tag. 😅 

![image](https://user-images.githubusercontent.com/6510020/125424996-94f182c2-d8fd-43af-abc7-6d2db47df934.png)

**Reply by wwmm** [reply]
I thought about that but I was not sure about what to do. Pulseaudio apps work in PipeWire and can be processed in EasyEffects. So the Pulseaudio tag is not entirely out of place.

**Comment by csehszlovakze** [comment]
I have an alternative and less confusing proposition. @wwmm , please bring back the pulseeffects repo at the last non-pipewire version and make it read-only so those of us who don't use pipewire at all can find it easier and fork it if needed. Right now, googling pulseeffects shows this repo under the old name and the rename was only visible here on github, and I had to dig for a while to see it was renamed entirely and not hacked or something.

**Reply by wwmm** [reply]
>please bring back the pulseeffects repo at the last non-pipewire version and make it read-only so those of us who don't use pipewire at all can find it easier and fork it if needed.

But if I do that I bet that there will be people asking to make the redirection working again. I doubt we will please everybody. The long discussion about the renaming has already made that clear. And as the legacy version for pulseaudio https://github.com/wwmm/easyeffects/tree/pulseaudio-legacy might still need update for critical bugs and that will be done here and not in the read-only repository people may start to question why the read-only repository exist. It is better to keep things as it is.
